src/types/rest/request/trade.ts

```diff
@@ -33,7 +33,7 @@ export interface AlgoLongHistoryRequest {
   limit?: string;
 }
 
-export interface AlgoOrderRequest {
+export interface PlaceAlgoOrderRequest {
```

Comment:
Renaming this is a breaking change (for anyone already importing & using `AlgoOrderRequest`), it's also now the only "request" interface prefixed with "Place". Please revert this. The rest looks good to me, thanks for the PR!

Reply:
You're right, reverted :)

Reply:
Thanks! The rest looks good, if you're finished, please also increase the version number in the package.json file - it'll make the release process easier :)

Reply:
Done ;) thanks for creating the package, am using it almost daily. Happy to contribute.
Ready for merge 🥳

Reply:
Glad it's helping! And thanks again for the PR. Will be on npm shortly.